packages/storage-plus/src/indexed_map.rs

```diff
@@ -196,6 +196,22 @@ where
     }
 }
 
+#[cfg(feature = "iterator")]
+impl<'a, K, T, I> IndexedMap<'a, K, T, I>
```

Comment:
I don't see any tests for these.

Reply:
There are no tests because they are the ranges for the base maps. But I will add them, it's more robust and maintainable.